src/overlays/gamestates/ovl_daytelop/z_daytelop.c

```diff
@@ -1,22 +1,247 @@
 /*
  * File: z_daytelop.c
  * Overlay: ovl_daytelop
- * Description:
+ * Description: "Dawn of ... day" screen
  */
 
 #include "z_daytelop.h"
+#include "static/daytelop_static/daytelop_static.h"
+#include "static/icon_item_gameover_static/icon_item_gameover_static.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_daytelop/Daytelop_Update.s")
+// unused
+UNK_TYPE D_808158E0[] = {
```

Comment:
Is this a texture?

Reply:
No, it isn't. it is way to sparse.
I checked it on Texture64 just to be sure anyways, and it doesn't look like one either